Corona-Warn-App/src/main/java/de/rki/coronawarnapp/notification/NotificationHelper.kt

```diff
@@ -194,19 +199,39 @@ object NotificationHelper {
      * @param pendingIntent: PendingIntent
      */
     fun sendNotification(
-        title: String,
+        title: String = CoronaWarnApplication.getAppContext().getString(R.string.notification_name),
```

Comment:
This default value should not be set

Reply:
Why? This way we make sure that there is no notification without title.